### ISSUE TITLE
Determine Overall Direction

### ISSUE THREAD

**Issue by loganfranken** [ucsb-wsg/web-accessibility#7]
I think we need to decide on an overall direction for rewriting this section:

One particular line caught my eye (https://github.com/ucsb-wsg/web-accessibility/blob/master/web-accessibility-guidelines.md): "The following guidelines are summarized from the W3C Content Accessibility Guidelines version 2.0 and Section 508 of the Rehabilitation Act."

On top of that, there are a lot guidelines that aren't necessarily accessibility-specific; they are just general tips for building a modern, maintainable website and could be strewn across the Web Guide.

To that end, I'm wondering if it would be best to take the following route:
1. Ensure all applicable guidelines from this guide are represented elsewhere in the Web Guide
2. Point to the [WCAG 2.0 Guidelines](https://www.w3.org/TR/WCAG20/) in the Web Guide's Accessibility section

In other words, we would effectively drop this separate Web Accessibility guide altogether.

I think this flows nicely with the idea that accessibility is a top concern that needs to be threaded throughout the development of a site; it's not an issue that can be addressed in an isolated document.

**Comment by UCSB-jcolo** [comment]
Hey Logan,
Sounds like a good approach to the general concepts.
Very true that an isolated document cant address the guidelines rather they need to be woven throughout..
Sincerely,
James

**Comment by loganfranken** [comment]
As far as I can tell, based on discussions within the Web Standards Group, we've decided to move forward with removing this PDF and spreading the content across the guide. Closing this issue.
